.github/dependabot.yml

```diff
@@ -4,16 +4,13 @@ updates:
     directory: "/"
     schedule:
       interval: daily
-      time: "11:00"
     open-pull-requests-limit: 10
   - package-ecosystem: gomod
     directory: "/"
     schedule:
       interval: weekly
     open-pull-requests-limit: 10
     reviewers:
-      - liamsi
-      - tzdybal
-      - Wondertan
+      @celestiaorg/execution-environment
     labels:
-      - T:dependencies
\ No newline at end of file
+      - T:dependencie
```

Comment:
[nit] Shouldn't this be dependency not dependencie

Reply:
uh, oh, the missing `s` on the end, accidentally removed `s`. good catch :+1:  fixing...